main.py

```diff
@@ -1,10 +1,51 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_info in players.items():
+        race = player_info["race"]
+        skills = race["skills"]
+        guild = player_info["guild"]
+
+        if not Race.objects.filter(name=race["name"]).exists():
+            Race.objects.create(
+                name=race["name"],
+                description=race["description"],
+            )
+        current_race = Race.objects.get(name=race["name"])
```

Comment:
can you optimize it? At least, you can make `race = Race.objects.create()...`. 
Then, you don't have to make an additional request to the database to get that value